bundle/tests/scorecard/kuttl/test-localdb/01-assert.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app-a-db
+  namespace: test-localdb
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/cloudservices/postgresql-rds:12-9ee2984
+          resources:
+            limits:
+              cpu: 400m
+              memory: 1Gi
+            requests:
+              cpu: 30m
+              memory: 512Mi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app-b-db
+  namespace: test-localdb
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/cloudservices/postgresql-rds:10-9ee2984
```

Comment:
Should we be asserting that the 'defaults' were used here for resources on app B's DB? I'm looking at https://github.com/RedHatInsights/clowder/pull/584/files#diff-5df7ec685a29a1eed6f24d8f66cf2129089eff23408cff9e91e550aa611f7a0fR113

Reply:
Yeah that makes sense. I'll add a test for defaults on app b.